### PR TITLE
fix: disable More button for multiple sources

### LINKED ISSUE #11760: 2.6 RC1: sync status and sync result "More" buttons do nothing in multi-source apps
Checklist:

**Describe the bug**

If I look at a multi-source Application's main page, "More" buttons for sync status and sync result do nothing.

**To Reproduce**

Use this app: 

```yaml
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: test
  namespace: argocd
spec:
  destination:
    namespace: default
    server: https://kubernetes.default.svc
  project: default
  sources:
  - ref: elastic
    repoURL: https://github.com/crenshaw-dev/values-file-repo
  - helm:
      valueFiles:
      - $elastic/values-prod.yaml
    path: helm-guestbook
    repoURL: https://github.com/argoproj/argocd-example-apps
    targetRevision: HEAD
```

View the app in the UI and go to the app summary page. Click one of the "More" buttons.

**Expected behavior**

I'm not sure what's wrong here. They should work.

**Screenshots**

image

**Version**

2.6.0-rc1

### PR DESCRIPTION
**Pull request by ishitasequeira**
Signed-off-by: ishitasequeira <ishiseq29@gmail.com>

Fixes #11760 

**Screenshots for UI changes:**
- For apps with multiple sources:

image

- For apps without multiple sources:

image

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [x] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: ui/src/app/applications/components/application-status-panel/application-status-panel.tsx [hunk 1]
```diff
@@ -30,12 +30,13 @@ const sectionLabel = (info: SectionInfo) => (
...
 
-const sectionHeader = (info: SectionInfo, onClick?: () => any) => {
+const sectionHeader = (info: SectionInfo, hasMultipleSources: boolean, onClick?: () => any) => {
     return (
         <div style={{display: 'flex', alignItems: 'center'}}>
             {sectionLabel(info)}
             {onClick && (
-                <button className='argo-button argo-button--base-o argo-button--sm application-status-panel__more-button' onClick={onClick}>
+                <button className='argo-button argo-button--base-o argo-button--sm application-status-panel__more-button' onClick={onClick} disabled={hasMultipleSources}>
+                    {hasMultipleSources && (helpTip("More button is not supported for apps with multiple sources"))}
```
Comment (crenshaw-dev) [inline ui/src/app/applications/components/application-status-panel/application-status-panel.tsx:39]:
```suggestion
                    {hasMultipleSources && (helpTip("Sync status details are not currently available for apps with multiple sources."))} 
```

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]
Can we add some space between the word "MORE" and the info icon?

File: ui/src/app/applications/components/application-status-panel/application-status-panel.tsx (hunk 1 above)
Comment (ishitasequeira) [reply ui/src/app/applications/components/application-status-panel/application-status-panel.tsx:39]: The More button is same for `Sync Status` and `Sync Result`. Thus, kept it same as "More button" for it to work in both cases.

Also `currently available` states that the details are expected to become available in some time or so, which is not the case.

I think below change might be better.. Let me know wdyt..

```
More details are not currently supported for apps with multiple sources.
```

**Review by crenshaw-dev — APPROVED** [review]
Thanks, @ishitasequeira!

**Comment by crenshaw-dev** [comment]
Cherry-picked onto release-2.6 for 2.6.0-rc6.

**Pull request merged**